.travis.yml

```diff
@@ -39,7 +39,7 @@ script:
   - ROOT_URL=open-event-frontend ember build -prod
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
+  - 'bash <(curl -s https://codecov.io/bash)'
```

Comment:
what?

Reply:
The script was not getting executed before. Mow the script is getting execute and also after every build a codecov logo and detail comes on Travis Logs. But why it is not producing any results?

Reply:
If you can tsll of there is anything more to add, it would be great

Reply:
Why does wrapping it in quotes work?

Reply:
I dont know, Just read somewhere. Although in worked in CI why its not generating report

Reply:
@iamareebjamal Any thing on this ?